handbook/unity.adoc

```diff
@@ -0,0 +1,160 @@
+
+=== Key idea involved in distributed spatial diversity
+
+Multiple communication nodes deployed at spatially distinct locations can receive independent copies of the same information. This group of receiver nodes can act as a combined spatial diversity receiver when they cooperate by sharing copies of information. We term this distributed spatial diversity. This framework is implemented in UnetStack as an agent and is referred to as Unity.
+
+[[fig_unityoverview]]
+.Cooperating receivers exploiting spatial diversity.
+image::unity-overview.png[]
+
+An illustration of the general overview of such a receiver system is shown in <<fig_unityoverview>> and few terminologies might help for further discussion:
+
+* *Transmitter*: A node which transmits information
+
+* *Main receiver*: A node that acts as the main receiver in a group of receiver nodes. It is the main receiver’s responsibility to decode the information.
+
+* *Assisting receiver*: A node that acts as an assisting receiver in a group of receiver nodes. It is the assisting receiver’s responsibility to forward the relevant information to the main receiver.
+
+The assisting receiver nodes are in cahoots with the main receiver to cooperatively share the information. This sharing of information usually happens over a short-range high-speed wired or wireless link (e.g., WiFi, Ethernet). Although, nothing stops one from using a different technology for sharing information.
+
+=== Immediate practical advantage
+
+Consider a case where you are on a ship with a modem deployed and you’ve been receiving status updates from a deployed AUV every few minutes. The AUV moves into an area where the connectivity is poor, and you can no longer successfully receive the status reports. We’ve all experienced being in that situation where the underwater communication link breaks, haven’t we?
+
+If you have a second modem available on the ship, you deploy it from another part of the ship. Or maybe there is a gateway buoy deployed nearby with a modem, and you can connect to its modem. Either way, the distributed spatial diversity technique (Unity) magically uses the information from both modems to recover connectivity to the AUV!
+
+=== How to use Unity in UnetStack3 ?
+
+Unity is available as a {url-editions}[premium] agent and requires UnetStack 3.1 or higher. Configuring and using `Unity` agent to exploit spatial diversity is easy with just two simple steps:
```

Comment:
I'd suggest following the same style as we use for `ECLink` on [this page](https://unetstack.net/handbook/unet-handbook_single_hop_links.html).